src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -303,27 +303,16 @@
 
 (defn message-content-wrapper
   "Author, userpic and delivery wrapper"
-  [{:keys [last-in-group? timestamp-str timestamp deleted? deleted-undoable-till
-           deleted-for-me? deleted-for-me-undoable-till pinned from chat-id]
+  [{:keys [last-in-group? deleted? deleted-for-me? timestamp
```

Comment:
could you please rollback this change? see [src/status_im2/contexts/chat/messages/content/deleted/view.cljs](https://github.com/status-im/status-mobile/pull/14573/files#diff-f19099d11cab857c4ab6061a65d7054b139f4e5feebd3d01860382f554cdd7c4)

Reply:
will do

Reply:
done